tutorials/e2e-huggingface-summarization/src/run_summarization/baseline.py

```diff
@@ -0,0 +1,62 @@
+import nltk
+from nltk.tokenize import sent_tokenize
+from datasets import load_dataset, load_metric
+import os
+import argparse
+import logging
+from azureml.core import Run
+import mlflow
+
+
+logger = logging.getLogger(__name__)
+nltk.download("punkt")
+rouge_score = load_metric("rouge")
```

Comment:
why do we need to have this variable here? couldn't we have it in main?

same for logger and for nltk.download() actually.

Reply:
moved to main